### PR TITLE
ADCM-6115: Implement Service object

### PR DESCRIPTION
**Pull request by DanBalalan**

### CODE REVIEW

File: adcm_aio_client/core/objects/_base.py [hunk 1]
```diff
@@ -48,7 +49,8 @@ def _construct_child[Child: "InteractiveChildObject"](
     def _clear_cache(self: Self) -> None:
         for name in self._delete_on_refresh:
             # works for cached_property
-            delattr(self, name)
+            with suppress(AttributeError):
```
Comment (Sealwing) [inline adcm_aio_client/core/objects/_base.py:52]: please leave a comment for suppress

as I understand it's for cases when value isn't cached, but it's a thing easy to forget

File: adcm_aio_client/core/objects/_common.py [hunk 2]
```diff
@@ -2,13 +2,20 @@
...
 from adcm_aio_client.core.objects._base import AwareOfOwnPath, WithRequester
+from adcm_aio_client.core.types import ADCMEntityStatus
 
 
 class Deletable(WithRequester, AwareOfOwnPath):
     async def delete(self: Self) -> None:
         await self._requester.delete(*self.get_own_path())
 
 
+class HasStatus(WithRequester, AwareOfOwnPath):
```
Comment (Sealwing) [inline adcm_aio_client/core/objects/_common.py:13]: maybe `WithStatus` as others?

File: adcm_aio_client/core/objects/_base.py (hunk 1 above)
Comment (DanBalalan) [reply adcm_aio_client/core/objects/_base.py:52]: done

File: adcm_aio_client/core/objects/_common.py (hunk 2 above)
Comment (DanBalalan) [reply adcm_aio_client/core/objects/_common.py]: renamed

**Review by Sealwing — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Review by a-alferov — DISMISSED** [review]

**Review by a-alferov — APPROVED** [review]

**Review by Starovoitov — APPROVED** [review]

**Pull request merged**